vcd/resource_vcd_vm_internal_disk_test.go

```diff
@@ -0,0 +1,479 @@
+// +build vapp vm ALL functional
+
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccVcdVmInternalDisk(t *testing.T) {
+	// This test requires access to the vCD before filling templates
```

Comment:
I don't see the reason the test needs vCD access before filling templates. Since we are creating the VDC, we don;'t need to check whether the VDC has fast provisioning.

Reply:
removed